src/main/webapp/app/app.module.ts

```diff
@@ -62,5 +64,6 @@ import { ArtemisSharedComponentModule } from 'app/shared/components/shared-compo
         LoadingNotificationComponent,
     ],
     bootstrap: [JhiMainComponent],
+    providers: [MatDialog],
```

Comment:
Why does this have to be added here? Please try to remove this using lazy imports if possible.

Reply:
Removed